### DISCUSSION TITLE
Update GDAL>=3.0 / PROJ>=6.0 performances issue

### DISCUSSION THREAD

**Discussion by fschulte-eiwa** [cogeotiff/rio-tiler#375] [Q&A] [answered]
Hi any updates regarding this issue? The related issue https://github.com/OSGeo/gdal/issues/3470 has been closed recently.

I followed the recommendation of "rasterio==1.1.8" but get this error:

"rasterio.errors.RasterioIOError: '/vsis3/***/mosaic_sample_cog.tif' does not exist in the file system, and is not recognized as a supported dataset name."

Couldn't find resolutions for the issue in that rasterio version.

With the latest rasterio version it works so I'm no sure if it's a AWS credentials configuration missing parameters for GDAL 2.4 or something else.

**Comment by kylebarron** [comment, marked as answer]
GDAL 3.3 will likely help, but there shouldn't be an issue loading from S3 with earlier versions either. You should check that you have access to the bucket and have set required env variables if needed, e.g. if requester pays.

**Reply by fschulte-eiwa** [reply]
Out of nowhere a new error appeared:
"rasterio.errors.RasterioIOError: CURL error: error setting certificate verify locations:   CAfile: /etc/pki/tls/certs/ca-bundle.crt   CApath: none"

SSL Certs was the problem (https://github.com/mapbox/rasterio#ssl-certs)

"export CURL_CA_BUNDLE=/etc/ssl/certs/ca-certificates.crt" solved that error and apparently the first one too.

Thanks

**Reply by kylebarron** [reply]
Yes on certain environments like AWS you need to set the cert path
